benchmarks/pom.xml

```diff
@@ -96,15 +96,30 @@
                                 </transformer>
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
                             </transformers>
+
                             <filters>
+                                <filter>
+                                    <!--
+                                        Shading signed JARs will fail without this.
+                                        http://stackoverflow.com/questions/999489/invalid-signature-file-when-attempting-to-run-a-jar
+                                    -->
+                                    <artifact>*:*</artifact>
+                                    <excludes>
+                                        <!-- Exclude module-info.class since it breaks strong encapsulation. -->
+                                        <exclude>module-info.class</exclude>
+                                        <exclude>META-INF/**</exclude>
+                                        <exclude>LICENSE</exclude>
+                                        <exclude>codegen-resources/**</exclude>
+                                    </excludes>
+                                </filter>
                             </filters>
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
             <plugin>
                 <artifactId>maven-deploy-plugin</artifactId>
-                <version>3.0.0-M1</version>
+                <version>2.8.1</version>
```

Comment:
This should remain on `3.0.0-M1`, as we target Maven 3.6.3 in our build pipeline. The `2.x` releases are for Maven 2 (https://maven.apache.org/plugins/maven-deploy-plugin/, https://issues.apache.org/jira/browse/MNG-6169)

Reply:
Removed maven deploy plugin since we don't use it.